components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/ChallengeQuestionProcessor.java

```diff
@@ -228,7 +233,9 @@ public UserChallengesDTO getUserChallengeQuestion(String userName, int tenantId,
 
         } catch (Exception e) {
             String msg = "No associated challenge question found for the user";
```

Comment:
Any reason for catching generic exception ? and why logging the exception only in debug mode ?